CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 5.5.3
+* Fixed range select issues with selectedItems increasing with duplicates
+* Simplified itemComparator to just itemKey: string
+
+
```

Comment:
I'm assuming this is b/c the previous build failed and so the bump commit wasn't made?